### ISSUE TITLE
Chunk loaded + unloaded message incorrect

### ISSUE THREAD

**Issue by pcmaster160** [CCM-Modding/Death-Timer#15]
Recently I have found that when I am under 30m away from a deathpoint it can say the chunk is unloaded and not let the timer go down. I am on far render distance and can actually see the chunk where I died in to confirm it is loaded.

I am using optifine multithreaded chunkloader, that may be the problem.

**Comment by dries007** [comment]
The  multicore could cause the problem, can you try without?

**Comment by pcmaster160** [comment]
I could not reproduce it since, I've looked at your code briefly and it seems to be right.

**Comment by dries007** [comment]
:p I have a another issue I can't explain from looking at the GitHub page. (I can't use eclipse right now) #13 & #14.
Feel free too look over that code and suggest fixes, @Claycorp, you can implement changes.

I'll be back in business +- 2013-07-15

**Comment by CaptainShadows** [comment]
or me :p

**Comment by CaptainShadows** [comment]
wrong button :P

**Comment by OniBait** [comment]
Is that in SSP or SMP? If it is SMP then it is very possible the chunk is unloaded on the server but the client still displaying it.

**Comment by dries007** [comment]
no, the server sends a packet to the client if the chunk (un)loads. That is supposed to keep all of the clients up to date.

**Comment by pcmaster160** [comment]
I was able to reproduce it without optifine installed and I'm pretty sure I figured out why it happens. The chunk loads/unloads when the game is paused and then death timer is out of sync with the chunk load status.

I was able to pause the game after I died but before I respawned and it said the chunk was loaded even though i was 4000+ blocks away, and my stuff was there 15 minutes after i died (so it truly wasn't loaded).
